content/en/database_monitoring/guide/connect_dbm_and_apm.md

```diff
@@ -46,6 +46,9 @@ Data privacy
 |          | [Npgsql][16]                   | {{< X >}}   |             |
 |          | [MySql.Data][17]               |             | {{< X >}}   |
 |          | [MySqlConnector][18]           |             | {{< X >}}   |
+| **PHP**  [dd-trace-php][19] >= 0.86.0 ||             |             |
```

Comment:
0.85.0 you mean? (just the restricting is going to be 0.86.0)

Reply:
I'd rather say `0.86.0` as it doesn't change anything and we'll be closer to the RFC with `0.86.0`